qiskit/aqua/operators/legacy/common.py

```diff
@@ -384,7 +384,7 @@ def evolution_instruction(pauli_list, evo_time, num_time_slices,
     return qc.to_instruction()
 
 
-def commutator(op_a, op_b, op_c=None, threshold=1e-12):
+def commutator(op_a, op_b, op_c=None, sign=-1, threshold=1e-12):
```

Comment:
I am sure the Bosonic PR has a change where this is now a boolean. I guess we will need to be careful with merging where it seems we have the same files.

Reply:
yes we need to merge this first and then the bosonic